packages/ses-ava/src/ses-ava-test.js

```diff
@@ -47,6 +45,45 @@ const logErrorFirst = (func, args, name, logger = defaultLogger) => {
   }
 };
 
+/**
+ * @param {Assertions} originalT
+ * @param {Logger} logger
+ * @returns {Assertions}
+ */
+const wrapAssertions = (originalT, logger) => {
+  /**
+   * Inherits all methods from the originalT that it does not override.
+   * Those that it does override it wraps behavior around a `super` call,
+   * and so uses concise method syntax rather than arrow functions.
+   *
+   * See TODO comment on cast below to remove the `{unknown}` type declaration.
+   *
+   * @type {unknown}
+   */
+  const newT = harden({
+    __proto__: originalT,
+    fail(message) {
+      const err = new Error(message);
+      logger('FAILED by t.fail', err);
+      return super.fail(message);
```

Comment:
Today I learned that `super` works without the enclosing `class`.

Reply:
This actually makes it more E-like, even though E did not provoke the idea. But still, E did help corroborate that it is a good idea.